spec/test_app/app/controllers/basechurch/v1/posts_controller_spec.rb

```diff
@@ -107,17 +107,17 @@
     context 'with an authenticated user' do
       before do
         request.headers['Content-Type'] = 'application/vnd.api+json'
-        request.headers['X-User-Email'] = logged_user.email
-        request.headers['X-User-Token'] = logged_user.session_api_key.access_token
+        request.headers['X-User-Email'] = user.email
+        request.headers['X-User-Token'] = user.session_api_key.access_token
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.